test/extensions/INTEL/SPV_INTEL_fpga_latency_control/IntelFPGALatencyControl.ll

```diff
@@ -0,0 +1,75 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_fpga_latency_control -o %t.spv
+; RUN: llvm-spirv %t.spv -to-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; CHECK-SPIRV: 2 Capability FPGALatencyControlINTEL
```

Comment:
No need to check number of words, until the test addresses big constants translation for example
```suggestion
; CHECK-SPIRV: Capability FPGALatencyControlINTEL
```

Reply:
Removed the check. thanks.